### DISCUSSION TITLE
Sentry in preact application

### DISCUSSION THREAD

**Discussion by tompol1123581321** [preactjs/preact#3382] [Q&A]
Hello I am trying to add sentry to my preact app and I am running into some issues using react-compat and classics '@sentry/react' init function. If anyone has any experience, ideas or points of reference on how to best solve this I would be glad to hear from you.

**Comment by marvinhagemeister** [comment]
> I am running into some issues

Can you elaborate on that? Is there an error logged to the console that would aid with figuring what's happening?
